tests-e2e/core/core_suite_test.go

```diff
@@ -23,3 +26,24 @@ func TestCore(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Core Suite", reporterConfig)
 }
+
+func EnsureCleanSlate() error {
```

Comment:
I think `EnsureCleanSlate()` and `GetE2ETestUserWorkspaceKubeClient()` can/should live in the `fixture` package, since they are fixture-adjacent (e.g. they are code that supports writing tests), and so that they can be called by other packages besides `tests-e2e/core`. Any objections?

Reply:
No objects, I just put it there cause it was just being used in tests-e2e/core; But refactored it now to work on fixture package.